config/gasSchedule.go

```diff
@@ -103,11 +105,12 @@ func CreateGasConfig(gasMap GasScheduleMap) (*GasCost, error) {
 		return nil, err
 	}
 
-	dynamicStorageLoadParams := &DynamicStorageLoad{}
-	err = mapstructure.Decode(gasMap["DynamicStorageLoad"], dynamicStorageLoadParams)
+	dynamicStorageLoadUnsigned := &DynamicStorageLoadUnsigned{}
+	err = mapstructure.Decode(gasMap["DynamicStorageLoad"], dynamicStorageLoadUnsigned)
 	if err != nil {
 		return nil, err
 	}
+	dynamicStorageLoadParams := convertFromUnsignedToSigned(dynamicStorageLoadUnsigned)
 
```

Comment:
we might add here a check that we will be sure that all resulted values will not be 0. And not just 0 but higher or equal to the coefficients.MinGasCost.

Reply:
Added.